docs/_data/importMap.cjs

```diff
@@ -34,6 +34,7 @@ const PFE_DEPS = [
   '@patternfly/pfe-core/functions/debounce.js',
   '@patternfly/pfe-core/functions/deprecatedCustomEvent.js',
   '@patternfly/pfe-core/functions/random.js',
+  '@lrnwebcomponents/code-sample',
```

Comment:
Adding this comment to inform that I'll be approving this PR and will resolve this issue pending @marionnegp design approval

Reply:
Thanks @brianferry. @marionnegp if  you're good, resolve this conversation to automerge. IMO, it's ok to merge this without first providing the video icon, we can open an new issue to deal with that.

Reply:
@marionnegp in the interest of progress, I'll merge now. If you have further review, it's very much welcome, but open a new issue. Thanks.